media_kit/lib/src/libmpv/player.dart

```diff
@@ -1124,6 +1148,41 @@ class Player extends PlatformPlayer {
       calloc.free(name);
       calloc.free(value);
     }
+    {
+      final name = 'demuxer-max-bytes'.toNativeUtf8();
+      final value = (32 * 1024 * 1024).toString().toNativeUtf8();
+      _libmpv?.mpv_set_property_string(
+        result,
+        name.cast(),
+        value.cast(),
+      );
+      calloc.free(name);
+      calloc.free(value);
+    }
+    {
+      final name = 'demuxer-max-back-bytes'.toNativeUtf8();
+      final value = (32 * 1024 * 1024).toString().toNativeUtf8();
+      _libmpv?.mpv_set_property_string(
+        result,
+        name.cast(),
+        value.cast(),
+      );
+      calloc.free(name);
+      calloc.free(value);
+    }
+    {
+      if (Platform.isAndroid) {
+        final name = 'ao'.toNativeUtf8();
+        final value = 'opensles'.toNativeUtf8();
```

Comment:
Using OpenSL ES for audio output on Android. The JNI based audiotrack was prone to crashes.